sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.0.0 (2020-10-30)
```

Comment:
Can you leave the date as `(unreleased)` for now?
We'll populate it when needed.

Reply:
Fixed.